server/lib/host-metrics.js

```diff
@@ -258,27 +259,43 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
   return orderBy(mergedTimeSeries, 'date');
 }
 
-export async function getTotalMoneyManagedTimeSeries(host, { startDate, endDate, timeUnit } = {}) {
+export async function getTotalMoneyManagedTimeSeries(
+  host,
+  { startDate, endDate, collectiveIds = null, timeUnit } = {},
+) {
   const hostedCollectives = await host.getHostedCollectives();
-  const ids = hostedCollectives.map(c => c.id);
+  if (!collectiveIds) {
```

Comment:
If you only need `hostedCollectives` when `collectiveIds` is falsy, then only fetch it when that's the case, not before.

Reply:
Ops, seems I've missed this one. Corrected it.